tests/nexus_workflow_test.go

```diff
@@ -772,7 +772,7 @@ func (s *ClientFunctionalSuite) TestNexusOperationAsyncCompletionErrors() {
 	s.NoError(err)
 
 	s.T().Run("ConfigDisabled", func(t *testing.T) {
```

Comment:
use `s.Run()` instead of `s.T().Run()`, then `s.T()` will return the subtest `t` in the test, and you can safely call `s.T()` from `overrideDynamicConfig`

Reply:
Oh that was the issue! Thanks! I tried removing t argument from OverrideDCValue() and reverted that because of this test failing.